frontend/src/workspace/globals/globals-themes.js

```diff
@@ -37,13 +37,13 @@ const nodeHoverShadow = {
 const menuBackground = {
     classic: 'white',
     dusk: 'white',
-    dark: '#1c170f',
+    dark: '#adadad',
 }
 
 // FOR NOW: Color of the value node on the menu value button.
 const valueMenuColor1 = "#F2937C";
 const valueMenuColor2 = "#ffa931";
-const valueMenuColor3 = "#ffa931";
+const valueMenuColor3 = "#463f36";
```

Comment:
What does this change?

Reply:
there are three types of values, I guess. this may have come from the updated colors for the value nodes (static, time-based, mouse-based, I think).